common/common.go

```diff
@@ -121,7 +121,7 @@ func AppTreeStoreIDPath(root string, appName types.ACName) string {
 }
 
 // AppImageManifestPath returns the path to the app's ImageManifest file
-func AppInfoImageManifestPath(root string, appName types.ACName) string {
+func AppImageManifestPath(root string, appName types.ACName) string {
```

Comment:
The directory where it is found is actually called `appinfo`. Could also just change the comment. I assume you're going with AppImageManifestPath instead of AppInfoImageManifestPath because it's shorter.

LGTM, in any case.